R/scGate.R

```diff
@@ -167,13 +167,19 @@ RunScGateWithRhesusModels <- function(seuratObj, min.cells = 30, assay = 'RNA',
       Bcell.RM = 'Bcell',
       Tcell.RM = 'T_NK',
       NK.RM = 'T_NK',
-      Myeloid.RM = 'MoMacDC',
```

Comment:
I think you need to add all the new models on line 165 too.

Reply:
thanks for catching that!